h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -602,10 +610,10 @@ private ModelMetrics makeMetrics(Frame data, Frame originalData, boolean isTrain
    * @param _train training data in the form of matrix
    * @param _valid validation data (optional, can be null)
    */
-  final void doScoring(Frame _train, Frame _trainOrig, Frame _valid, Frame _validOrig) {
+  final void doScoring(Frame _train, Frame _trainOrig, CustomMetric trainCustomMetric, Frame _valid, Frame _validOrig) {
```

Comment:
Odd parameter naming convention. I don't know why but all existing parameter names start with underscore. Adding a new parameter, without underscore, leads to even more weird situation.

Reply:
good point, thanks for catching that - I will fix it